### PR TITLE
UT3 Scorpion Overall Improvements

### PR DESCRIPTION
**Pull request by HellDragon-HK**
This is everything I've changed with the Scorpion for now, which includes...
-Fixed boost sound on 32 bit
-Handling
-Sound Pack Update https://drive.google.com/open?id=1l7cC58tLhf-nAps3Dzk00FttPPchNaDP (includes all missing sounds and a temporary mix to solve a missing sound issue until done properly)
-Fire Damage Rate
-Speed
-Restored Projector
-How much vehicle is thrown around by weapon fire
-Exit points

### CODE REVIEW

**Comment by HellDragon-HK** [comment]
Sigh dang it, I had the driving pose almost perfect on this one, but either the stock model's feet go through the top or my preferred skaarj hybrid model's feet go through the floor, so this last update I found a happy medium which isn't perfect for either but the clipping isn't visible at least and it's not easy to see inside the Scorpion anyway so most probably won't notice how the stock model's legs float while the other goes slightly through front (not far enough to be seen though)

File: Classes/UT3Scorpion.uc [hunk 1]
```diff
@@ -2,6 +2,7 @@
  * Copyright © 2008, 2014 GreatEmerald
  * Copyright © 2008-2009 Wormbo
  * Copyright © 2012 100GPing100
+ * Copyright © 2017 HellDragon (Copyright seems too strong a word to me but if that's what you want)
```
Comment (GreatEmerald) [inline Classes/UT3Scorpion.uc]: No comments in copyright

File: Classes/UT3Scorpion.uc [hunk 2]
```diff
@@ -45,6 +46,14 @@ var IntBox BoostIconCoords, EjectIconCoords;
 var float LastBoostAttempt, SpeedAtBoost;
 var() float MinEjectSpeed;
 var int AirBoost;
+Var rotator FootDriveL,FootDriveR;
+Var rotator ArmDriveL,ArmDriveR;
+Var rotator ForeArmDriveL, ForeArmDriveR;
+Var rotator ThighDriveL,ThighDriveR;
+Var rotator CalfDriveL,CalfDriveR;
+Var rotator SpineDrive;
+Var rotator NeckDrive;
```
Comment (GreatEmerald) [inline Classes/UT3Scorpion.uc]: Again these need to be either local or their properties inside `defaultproperties`

File: Classes/UT3Scorpion.uc [hunk 3]
```diff
@@ -45,6 +46,14 @@ var IntBox BoostIconCoords, EjectIconCoords;
...
+var bool bAttachedDriver;
```
Comment (GreatEmerald) [inline Classes/UT3Scorpion.uc]: Again this is needless

File: Classes/UT3Scorpion.uc [hunk 4]
```diff
@@ -539,12 +548,72 @@ simulated event DrivingStatusChanged()
...
+}
+
 //=============================================================================
 // Default values
 //=============================================================================
 
 defaultproperties
 {
+
+    //DrawScale=1.2
```
Comment (GreatEmerald) [inline Classes/UT3Scorpion.uc]: Again check whether 1.0 is the actual default here

**Comment by HellDragon-HK** [comment]
It is from ONSRV
	DrawScale=1.0
	DrawScale3D=(X=1.0,Y=1.0,Z=1.0)

**Comment by GreatEmerald** [comment]
Note that since you've been working on this during both 2017 and 2018, your copyright in those files you edited during both years should read `2017-2018`.

**Comment by GreatEmerald** [comment]
You'll need to remove the binary upload from this pull request. That probably means that you'll need to redo it anyway and thus it's better to do after the binary purge.

**Comment by HellDragon-HK** [comment]
Redone in new request

**Pull request closed without merging**